components/style/properties/gecko.mako.rs

```diff
@@ -2393,8 +2398,18 @@ fn static_assert() {
     }
 
     #[allow(non_snake_case)]
-    pub fn reset__x_lang(&mut self, other: &Self) {
-        self.copy__x_lang_from(other)
+    pub fn set__moz_script_min_size(&mut self, v: longhands::_moz_script_min_size::computed_value::T) {
```

Comment:
I believe we can create these methods with something like 
```
${impl_absolute_length("_moz_script_min_size", "mScriptMinSize")}
```
Or we can append this dictionary with `"MozScriptMinSize": impl_absolute_length,`: https://dxr.mozilla.org/servo/source/components/style/properties/gecko.mako.rs#1107 This list is probably for more generic stuff but I don't have a preference.

Reply:
Thanks for review! I'll try the second solution!